Surface_mesher/include/CGAL/IO/output_surface_facets_to_polyhedron.h

```diff
@@ -23,17 +23,17 @@
 
 namespace CGAL {
 
-/// \deprecated Gets reconstructed surface out of a SurfaceMeshComplex_2InTriangulation_3 object.
+/// \deprecated Gets reconstructed surface out of a `SurfaceMeshComplex_2InTriangulation_3` object.
 ///
 /// This variant exports the surface as a polyhedron.
 /// It requires the surface to be manifold. For this purpose,
-/// you may call make_surface_mesh() with Manifold_tag or Manifold_with_boundary_tag parameter.
+/// you may call `make_surface_mesh()` with `Manifold_tag` or `Manifold_with_boundary_tag` parameter.
 ///
-/// @commentheading Template Parameters:
-/// @param SurfaceMeshComplex_2InTriangulation_3 model of the SurfaceMeshComplex_2InTriangulation_3 concept.
-/// @param Polyhedron an instance of CGAL::Polyhedron_3<Traits>.
+/// *Template Parameters:*
```

Comment:
Is this line not a bit overdone as doxygen will add a header **Template Parameters**